MainRepository/Program.cs

```diff
@@ -1,7 +1,13 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿using System.Text;
+// Reduce string
+Console.Write ("Enter the string: ");

```

Comment:
Add test cases that handles different scenarios. That helps in reviewing the code faster.